features/release.feature

```diff
@@ -367,6 +367,36 @@ MARKDOWN
           ASSET_TARBALL
           """
 
+  Scenario: Delete a release
+    Given the GitHub API server:
+      """
+      get('/repos/mislav/will_paginate/releases') {
+          json [
+            { url: 'https://api.github.com/repos/mislav/will_paginate/releases/123',
+              upload_url: 'https://api.github.com/uploads/assets{?name,label}',
+              tag_name: 'v1.2.0',
+              name: 'will_paginate 1.2.0',
+              draft: true,
+              prerelease: false,
+              assets: [
+                { url: 'https://api.github.com/repos/mislav/will_paginate/assets/9876',
+                  name: 'hello-1.2.0.tar.gz',
+                },
+              ],
+            },
+          ]
+      }
+
+      delete('/repos/mislav/will_paginate/releases/123') {
+        status 204
+      }
+      """
+    When I successfully run `hub release delete v1.2.0`
+    Then the output should contain exactly:
+      """
+      Deleting release related to v1.2.0...\n
```

Comment:
I think I'm gonna vote for that one. I'll push my updates for the rest of the fixes here in just a second.